packages/embark/src/lib/modules/parity/index.js

```diff
@@ -51,23 +61,41 @@ class Parity {
   }
 
   _getNodeState(err, version, cb) {
-    if (err) return cb({name: "Ethereum node not found", status: 'off'});
+    if (err) return cb({ name: "Ethereum node not found", status: 'off' });
 
     let nodeName = "parity";
     let versionNumber = version.split("-")[0];
     let name = nodeName + " " + versionNumber + " (Ethereum)";
-    return cb({name, status: 'on'});
+    return cb({ name, status: 'on' });
+  }
+
+  _doCheck(cb) {
+    const { rpcHost, rpcPort, wsRPC, wsHost, wsPort } = this.blockchainConfig;
+    if (wsRPC) {
+      return ws(wsHost, wsPort + 10, (err, version) => this._getNodeState(err, version, cb));
+    }
+    rpc(rpcHost, rpcPort + 10, (err, version) => this._getNodeState(err, version, cb));
```

Comment:
Again, you can remove the `+10`

Reply:
Removed the +10 so the node is pinged directly. Good catch.